 # Single depth reading.

Header header           # timestamp is the time the depth was measured
                         # frame_id is the location of the depth reading

 float64 depth     # Measurement of the Depth in meters

 float64 variance        # 0 is interpreted as variance unknown
